libs/payments/stripe/src/lib/stripe.client.ts

```diff
@@ -38,7 +38,7 @@ export class StripeClient {
   private readonly stripe: Stripe;
   constructor(private stripeConfig: StripeConfig) {
     this.stripe = new Stripe(this.stripeConfig.apiKey, {
-      apiVersion: '2022-11-15',
+      apiVersion: '2024-04-10',
```

Comment:
Considering this is set in 5&times;, should this be set as an ENV var or somewhere more global?
Or maybe wrap the `Stripe()` constructor:

<details>
<summary><kbd>git grep -n -B1 -A2 "2024-04-" | cat</kbd></summary>

```sh
libs/payments/stripe/src/lib/stripe.client.ts-40-    this.stripe = new Stripe(this.stripeConfig.apiKey, {
libs/payments/stripe/src/lib/stripe.client.ts:41:      apiVersion: '2024-04-10',
libs/payments/stripe/src/lib/stripe.client.ts-42-      maxNetworkRetries: 3,
libs/payments/stripe/src/lib/stripe.client.ts-43-    });
--
packages/fxa-admin-server/src/subscriptions/stripe.service.ts-36-    const stripe = new Stripe(stripeConfig.stripeApiKey, {
packages/fxa-admin-server/src/subscriptions/stripe.service.ts:37:      apiVersion: '2024-04-10',
packages/fxa-admin-server/src/subscriptions/stripe.service.ts-38-      maxNetworkRetries: 3,
packages/fxa-admin-server/src/subscriptions/stripe.service.ts-39-    });
--
packages/fxa-auth-server/lib/payments/stripe.ts-203-    this.stripe = new Stripe(config.subscriptions.stripeApiKey, {
packages/fxa-auth-server/lib/payments/stripe.ts:204:      apiVersion: '2024-04-10',
packages/fxa-auth-server/lib/payments/stripe.ts-205-      maxNetworkRetries: 3,
packages/fxa-auth-server/lib/payments/stripe.ts-206-    });
--
packages/fxa-auth-server/scripts/audit-orphaned-customers.ts-145-  const stripe = new Stripe(config.subscriptions.stripeApiKey, {
packages/fxa-auth-server/scripts/audit-orphaned-customers.ts:146:    apiVersion: '2024-04-10',
packages/fxa-auth-server/scripts/audit-orphaned-customers.ts-147-    maxNetworkRetries: 3,
packages/fxa-auth-server/scripts/audit-orphaned-customers.ts-148-  });
--
packages/fxa-auth-server/scripts/refund-unverified-accounts.ts-152-  const stripe = new Stripe(config.subscriptions.stripeApiKey, {
packages/fxa-auth-server/scripts/refund-unverified-accounts.ts:153:    apiVersion: '2024-04-10',
packages/fxa-auth-server/scripts/refund-unverified-accounts.ts-154-    maxNetworkRetries: 3,
packages/fxa-auth-server/scripts/refund-unverified-accounts.ts-155-  });
```
</details>